core/includes/install.core.inc

```diff
@@ -1454,6 +1450,341 @@ function install_select_language_form($form, &$form_state, $files) {
   return $form;
 }
 
+/**
+ * Provide a list of available translation files.
+ *
+ * @return array
+ *   An array of language codes.
+ */
+function install_get_available_translations($files) {
+  $available_translations = array();
+  $response = backdrop_http_request(INSTALL_AVAILABLE_TRANSLATIONS_URI);
```

Comment:
A timeout would also be useful for this request.

Reply:
Fixed